gensim/keras_integration/__init__.py

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2011 Radim Rehurek <radimrehurek@seznam.cz>
+# Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
+"""Keras wrappers for gensim.
```

Comment:
"Wrapper to allow gensim word2vec as input into Keras." is more clear. And in other docstrings and ipynbs too